YoCode/ProjectBuilder.cs

```diff
@@ -70,7 +70,7 @@ private void CheckBuildSuccess()
         {
             if(Output.Contains("is being used by another process"))
             {
-                ProjectBuilderEvidence.SetInconclusive(messages.BadPort);
+                ProjectBuilderEvidence.SetInconclusive("Could not project's build status. It is being used by another process");
```

Comment:
Typo here. "Could not project's build status" should be "Could not build the project"